mgnify_pipelines_toolkit/analysis/assembly/krona_txt_from_CAT_classification.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2025 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+from collections import Counter
+import csv
+
+TAXON_PREFIXES = {
+    5: "sk__",
+    6: "p__",
+    7: "c__",
+    8: "o__",
+    9: "f__",
+    10: "g__",
+    11: "s__",
+}
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Process TSV classification generated by CAT_pack add_names and write input file for Krona ktImportText"
+    )
+    parser.add_argument(
+        "-i", "--cat-output", help="Path to the input TSV file from CAT_pack add_names"
+    )
+    parser.add_argument("-o", "--krona-txt", help="Name of the output Krona TXT file")
```

Comment:
same thing here, I think you can just call the long form `--output` really, the help message gives a good description of what the output actually is